app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -19,7 +19,13 @@
 </h1>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <%= render ProviderInterface::StatusBoxComponent.new(application_choice: @application_choice) %>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column">
```

Comment:
I know there's nothing for the other third now, but I miss the `two-thirds` a bit. The lines feel very long

<img width="1044" alt="Screenshot 2020-02-18 at 17 13 40" src="https://user-images.githubusercontent.com/642279/74760260-090fc800-5272-11ea-8ac6-70372dca3874.png">

Reply:
Bringing two-thirds back